src/curated-corpus/components/ApprovedItemListCard/ApprovedItemListCard.tsx

```diff
@@ -26,27 +26,50 @@ interface ApprovedItemListCardProps {
    * An object with everything approved curated item-related in it.
    */
   item: ApprovedCuratedCorpusItem;
+
+  /**
+   * Optional boolean prop to show/hide the language icon
+   */
+  showLanguageIcon?: boolean;
+
+  /**
+   * Optional boolean prop to show/hide the "Rec." overlay
+   */
+  showRecommendedOverlay?: boolean;
 }
 
 export const ApprovedItemListCard: React.FC<ApprovedItemListCardProps> = (
   props
 ): JSX.Element => {
   const classes = useStyles();
-  const { item } = props;
+  const {
+    item,
+    showLanguageIcon = true,
+    showRecommendedOverlay = true,
+  } = props;
+
+  const isRecommendedOverlayVisible =
+    showRecommendedOverlay && item.status === CuratedStatus.Recommendation;
+
+  // prefixing the item's imageUrl with the pocket-image-cache url to format it to a size of 300x150
+  const formattedImageUrl =
+    `https://pocket-image-cache.com/300x150/filters:format(jpg):extract_focal()/`.concat(
+      item.imageUrl
+    );
```

Comment:
Using a hardcoded size of `300x150`, thoughts?

Reply:
I'd double it to cater for higher screen resolutions